src/experiments.js

```diff
@@ -364,6 +364,16 @@ export function getExperimentBranch(win, experimentName) {
     : null;
 }
 
+/**
+ * Returns an object containing all active experiment branches.
+ *
+ * @param {!Window} win Window context to check for experiment state.
+ * @return {?Object} contains all experiment branches and their ids.
+ */
+export function getActiveExperimentBranches(win) {
+  return win.__AMP_EXPERIMENT_BRANCHES ? win.__AMP_EXPERIMENT_BRANCHES : null;
```

Comment:
I saw we initiate `win.__AMP_EXPERIMENT_BRANCHES` whenever it is accessed. I'd prefer us always return an object here as well. So you don't need special check for `null` value.

Reply:
Done.